ci/scripts/run-benchmarks.sh

```diff
@@ -20,7 +20,7 @@ export PG_VERSION=15
 export GITHUB_OUTPUT=/dev/null
 export PGDATA=/etc/postgresql/$PG_VERSION/main/
 
-echo "port = 5432" >> $PGDATA/postgresql.conf
+echo "port = 5432" >> ${PGDATA}postgresql.conf
```

Comment:
It seems `PGDATA` has a trailing slash, but other path variables do not (e.g. WORKDIR does not)
We should choose and document a convention and make sure it is followed everywhere.

`pg_config` seems to return paths without the traling slash. Example:
```
INCLUDEDIR-SERVER = /usr/include/postgresql/13/server
LIBDIR = /usr/lib/x86_64-linux-gnu
PKGLIBDIR = /usr/lib/postgresql/13/lib
```
So, I am inclined to use that convention here.